src/front/app/features/Menu.js

```diff
@@ -24,3 +21,11 @@ Menu.propTypes = {
 };
 
 export default Menu;
+
+
+/* For release was deleted or changed data
```

Comment:
can we use TODO syntax for comments ? https://blog.jetbrains.com/webide/2012/10/managing-todo/